chapters/01_introduction/main.tex

```diff
@@ -1 +1,600 @@
-\chapter{Introduction}\label{sec:introduction}
\ No newline at end of file
+\chapter{Introduction}\label{sec:introduction}
+
+Operational Research (OR) is a field of study that makes use of mathematical
+and computational techniques to solve problems in a wide range of areas, such
+as healthcare, business, engineering, and the military.
+The core of OR has been to help decision makers make better decisions by
+providing them with the tools to analyse and understand the problem at hand and
+the possible solutions to it.
+Such mathematical tools include queueing theory, game theory, decision theory,
+statistical analysis, machine learning and many more.
+
+This thesis focuses on the application of OR to healthcare systems and in
+particular to the Emergency Department (ED) and Emergency Medical Services
+(EMS) interface.
+OR models have been used a lot in the past in conjunction with healthcare
+systems to better inform the decision-making of healthcare managers and
+policy makers.
+The use of OR in healthcare systems can been particularly useful in situations
+where improvements in the system are needed to cope with the demand for
+healthcare services.
```

Comment:
I think I'd swap the order of these two paragraphs.

Reply:
Will do. I don't think I have a preference